src/construction.h

```diff
@@ -2,18 +2,20 @@
 #ifndef CONSTRUCTION_H
 #define CONSTRUCTION_H
 
-#include "cursesdef.h" // WINDOW
-#include "enums.h" // point
 #include "string_id.h"
 
 #include <string>
 #include <set>
+#include <map>
+#include <vector>
 #include <functional>
 
 class JsonObject;
 typedef int nc_color;
 class Skill;
 struct requirement_data;
+struct WINDOW;
```

Comment:
If you need `WINDOW`, just include `cursesdef.h` - it contains the needed forward declaration.